src/TSTransformer/nodes/statements/transformImportEqualsDeclaration.ts

```diff
@@ -1,11 +1,20 @@
 import luau from "@roblox-ts/luau-ast";
 import { assert } from "Shared/util/assert";
 import { TransformState } from "TSTransformer";
+import { transformIdentifierDefined } from "TSTransformer/nodes/expressions/transformIdentifier";
 import { transformVariable } from "TSTransformer/nodes/statements/transformVariableStatement";
 import { createImportExpression } from "TSTransformer/util/createImportExpression";
 import { isSymbolOfValue } from "TSTransformer/util/isSymbolOfValue";
 import ts from "typescript";
 
+function transformEntityName(state: TransformState, node: ts.EntityName) {
+	return ts.isIdentifier(node) ? transformIdentifierDefined(state, node) : transformQualifiedName(state, node);
```

Comment:
Maybe put an assertNever call on the end here?

Reply:
I don't think it's necessary. They're useful in some places where definitions are expected to change in the future. I can't imagine EntityName ever changing.

Reply:
Hm. I think it can still be useful for version mismatch issues, trying to bail if a transformer went wrong somewhere.